src/views/compute/DeployVM.vue

```diff
@@ -275,6 +275,16 @@
                   </div>
                 </template>
               </a-step>
+              <a-step
+                :title="this.$t('enter BIOS setup')"
```

Comment:
`this.$t..` can be written as `$t(...). Instead of a string, can you use a key, we usually encourage the same key as the API arg/parameter if applicable. You can then define the key and the language translation in src/locales/en.json